### ISSUE TITLE
Issue: I want to turn off automatic hurricane casting for druid

### ISSUE THREAD

**Issue by blablabla12341234** [kolton/d2bot-with-kolbot#2820]
I am making a Fire elemental druid.  And when it automatically casts hurricane, it screws up the timing for fissure and armageddon.
how do i turn off hurricane??
thanks

**Comment by blablabla12341234** [comment]
Also, 2 bots, one wind druid and one fire elemental.
i still want the wind druid to auto cast hurricane,
but i dont want to fire druid to auto cast hurricane.
Heck if you can tell me for the fire druid, how to set it up to auto cast armageddon instead of hurricane, that would be great.

have like 2 separate file for each of my druids config scripts to reference.

**Comment by ARkaine2** [comment]
...\KolBot\D2BS\KolBot\Libs\Common\Precast.js

```
			if (me.getSkill(250, 0) && (!me.getState(144) || force)) {
				Skill.cast(250, 0); // Hurricane
			}
```

try to change it to:

```
			if (me.getSkill(249, 0) && (!me.getState(145) || force)) {
				Skill.cast(249, 0); // Armageddon
			}
```

**Comment by blablabla12341234** [comment]
Yes i found that as well. The issue is im running wind druid AND fire elemental druid in the same party.  If i change this to armageddon, my wind druid would be affected as well.
How can i set it up so the wind druid refers the original precast script, and the fire druid refers to. a copy paste of the original script with armegeddon, lets say names precast1.js

**Comment by blablabla12341234** [comment]
> ...\KolBot\D2BS\KolBot\Libs\Common\Precast.js
> 
> ```
> 			if (me.getSkill(250, 0) && (!me.getState(144) || force)) {
> 				Skill.cast(250, 0); // Hurricane
> 			}
> ```
> 
> try to change it to:
> 
> ```
> 			if (me.getSkill(249, 0) && (!me.getState(145) || force)) {
> 				Skill.cast(249, 0); // Armageddon
> 			}
> ```

Yes i found that as well. The issue is im running wind druid AND fire elemental druid in the same party. If i change this to armageddon, my wind druid would be affected as well.
How can i set it up so the wind druid refers the original precast script, and the fire druid refers to a copy paste of the original script with armegeddon, lets say nameD precast1.js

**Comment by blablabla12341234** [comment]
i guess noone knows where in the code you can reference another precast file

**Comment by blablabla12341234** [comment]
Someone out there must have a solution

**Comment by qnts** [comment]
@blablabla12341234 

In your fire druid config file, add `Config.PrecastForceArmageddon = true;`

In `Libs\Common\Precast.js`, change

```js
if (me.getSkill(250, 0) && (!me.getState(144) || force)) {
	Skill.cast(250, 0); // Hurricane
}
```
to
```js
if (!me.getState(144) || force) {
    if (Config.PrecastForceArmageddon && me.getSkill(249, 0)) {
        Skill.cast(249, 0); // Armageddon
    } else if (me.getSkill(250, 0)) {
        Skill.cast(250, 0); // Hurricane
    }
}
```

**Comment by blablabla12341234** [comment]
> @blablabla12341234
> 
> In your fire druid config file, add `Config.PrecastForceArmageddon = true;`
> 
> In `Libs\Common\Precast.js`, change
> 
> ```js
> if (me.getSkill(250, 0) && (!me.getState(144) || force)) {
> 	Skill.cast(250, 0); // Hurricane
> }
> ```
> 
> to
> 
> ```js
> if (!me.getState(144) || force) {
>     if (Config.PrecastForceArmageddon && me.getSkill(249, 0)) {
>         Skill.cast(249, 0); // Armageddon
>     } else if (me.getSkill(250, 0)) {
>         Skill.cast(250, 0); // Hurricane
>     }
> }
> ```

Thank you my friend this works perfectly

**Comment by qnts** [comment]
@blablabla12341234 glad it helped
